python-avd/pyavd/_eos_designs/schema/schema_fragments/digital_twin.schema.yml

```diff
@@ -28,21 +29,20 @@ keys:
           Settings for Digital Twin fabric devices.
         required: true
         keys:
-          # TODO: Enforce (in code) to be set (here or under node_config)
-          os_version:
+          act_os_version:
```

Comment:
so this single key will target every node type (including third-party and cloudvision) -> maybe we need to evolve the model as the node_config level key will not be present for these (we don't have third-party node_type_keys today)